versioned/persist/adapter/src/main/java/org/projectnessie/versioned/persist/adapter/spi/AbstractDatabaseAdapter.java

```diff
@@ -863,6 +886,7 @@ public final CommitLogEntry fetchFromCommitLog(OP_CONTEXT ctx, Hash hash) {
   protected abstract CommitLogEntry doFetchFromCommitLog(OP_CONTEXT ctx, Hash hash);
 
   @Override
+  @SuppressWarnings("MustBeClosedChecker")
   public Stream<CommitLogEntry> scanAllCommitLogEntries() {
```

Comment:
isnt this missing `@MustBeClosed` ?
because again we are adding additional cleanup logic to the returned stream

Reply:
the interface has it

Reply:
in the other comment you mention that annotations are not inherited... but for interfaces this works?
idk, i would learn torwards just adding them here as well to be on the safe side... but up to you.

Reply:
call-sites usually refer to the interface - so they "use" the annotation